### PR TITLE
Create a standard to lock the device screen

### PR DESCRIPTION
**Pull request by Miniplop**
@tychota Check this out :)

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/standard-orientation-ada/react-native/features/lock-device-orientation.mo.md">lock-device-orientation.mo.md</a>**: Seems you do not need a *Troubleshoot* part
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: react-native/features/lock-device-orientation.mo.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# [MO] Lock the device screen
+
+## Owner: [Kévin Jean](https://github.com/Miniplop)
+
+## Prerequisites
+- [ ] A running React Native app, preferably started with the generator
```
Comment (tychota) [inline react-native/features/lock-device-orientation.mo.md]: Les `- [ ]` passent pas sur gitbook. Tu peux mettre seulement `-`!

File: react-native/features/lock-device-orientation.mo.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Steps
+
+- Follow [this](https://github.com/yamill/react-native-orientation/blob/master/README.md) to install react-native-orientation
+
+> :warning: Do not forget to rebuild your app
+
+### Lock all the app pages
+
+> :warning: Do not lock the screen on react-native-camera or other native modules which needs screen rotation
```
Comment (tychota) [inline react-native/features/lock-device-orientation.mo.md]: lock the screen rotation

**Review by tychota — APPROVED** [review]
After changes !

**Pull request merged**